scripts/build_directory_md.py

```diff
@@ -1,17 +1,28 @@
 #!/usr/bin/env python3
 
+from __future__ import annotations
```

Comment:
This repo runs code on the current Python (3.11 this month, 3.12 next month) so this import is not required.